circle.yml

```diff
@@ -13,10 +13,32 @@ jobs:
           steps:
             - run: npm install
             - run: npm rebuild node-sass
-            - run: npm run test:ci
             - run: npm run build
+  test:
+    docker:
+      - image: circleci/node:12
+      - environment:
+          NODE_ENV: test
+          POSTGRES_USER: root
+          POSTGRES_PASSWORD:
+          POSTGRES_DB: nhs-virtual-visit-test
+        image: "circleci/postgres:12.2"
```

Comment:
Can you move this into a ```- image:``` similar to ```- image: circleci/node:12```

Reply:
So maybe the structure could be 
```- image: circleci/node:12
     - image: circleci/postgres:12.2
```

Reply:
Will take a crack at it. The docs I were following had the image indented which I assumed was doing some linking under the hood.

Reply:
Cool no worries :)

Reply:
I just tested moving it around and it causes a build error. The funky indentation is intentional 🤷

Reply:
Yaml has some very strange, I think ambiguous, syntax around objects :(